pinot-connectors/presto-pinot-driver/pom.xml

```diff
@@ -36,6 +36,7 @@
     <properties>
         <pinot.root>${basedir}/../..</pinot.root>
         <shadeBase>org.apache.pinot.\$internal</shadeBase>
+        <jdk.version>8</jdk.version>
```

Comment:
Does this still require code to be java 8 compatible?

Reply:
Here are some leftover works once folks(e.g. Uber) upgrade their build system from JDK 8. We can completely change the profile of the JDK8 build and call that Pinot deprecated JDK 8 build but can be released with JDK 8 binary.